consutil/main.py

```diff
@@ -31,7 +31,17 @@ def line():
 @click.argument('linenum')
 def clear(linenum):
     """Clear preexisting connection to line"""
-    click.echo("clear line linenum")
+    checkDevice(linenum)
+    linenum = str(linenum)
+
+    busyDevices = getBusyDevices()
+    if linenum in busyDevices:
+        pid, _ = busyDevices[linenum]
+        cmd = "sudo kill -15 " + pid
```

Comment:
I'd suggest to use `kill -SIGTERM $pid` or `kill -TERM $pid` here. More readable than `-15`